src/rp2/plugin/report/abstract_ods_generator.py

```diff
@@ -101,7 +101,10 @@ def _initialize_output_file(  # pylint: disable=too-many-branches
                 cls._fill_cell(legend_sheet, index + 2, 1, to_date if to_date != MAX_DATE else "non-specified", visual_style="transparent")
                 break
         if not method_cell_found:
-            raise Exception("Internal error: ODS template has no 'Accounting Method' cell in column 0 of Legend sheet")
+            local_accounting_method: str = _("Accounting Method")
+            raise Exception(
+                f"Internal error: ODS template has no 'Accounting Method' ({local_accounting_method}) cell in column 0 of Legend sheet ({legend_sheet_name})"
```

Comment:
This might not work too well in English, because `local_accounting_method` would equate to `'Accounting Method'`. Same consideration for `legend_sheet_name`. The recommended way to provide translations is to to just encapsulate the entire message string into _() and then provide a translation for it by following these instructions: https://github.com/eprbell/rp2/blob/main/README.dev.md#localization

Reply:
Adding a little context here: right now you're generating Japanese taxes in English, so translations won't have any effect (because generation language is English). If you were to generate Japanese taxes in Japanese, then it would be different. So my suggestion is: 
* if you're only doing Japanese taxes in English it's probably better to undo this change, 
* if you're also adding Japanese taxes in Japanese you can translate the entire message and optionally all other messages in RP2 (using instructions from the link above), but that's up to you.

Reply:
Well, I basically rewrote this error message for the localization test. The error is misleading in that it states it is looking for 'Accounting Method', but it is actually looking for '__test_Accounting Method' in the `template_tax_report_jp_kl.ods` file (for the localization test).

I'd like to do all the localization later when I have the time to link it all instead of doing it piecemeal. I can revert this for now if you think it will cause confusion.

Reply:
Oh, now I understand what you meant. Here's how I would do it then:
```
            raise Exception("Internal error: ODS template has no '{_(Accounting Method)}' cell in column 0 of '_(Legend)' sheet")
```